pkg/database/postgres_db_instance.go

```diff
@@ -39,23 +38,15 @@ func NewPostgresDBInstance(
 		DBInstance: DBInstance{dsn: dsnURL.String()},
 	}
 
-	var sourceConn gorm.Dialector
-	var replicaConn gorm.Dialector
-	sourceConn = postgres.Open(dsnURL.String())
+	conn := postgres.Open(dsnURL.String())
 
-	logURL := dsnURL
-	q := logURL.Query()
-	if q.Has("_key") {
-		q.Set("_key", "xxxxx")
-	}
-	logURL.RawQuery = q.Encode()
-	log.Infof("Using database %s", logURL.Redacted())
```

Comment:
hah, this is what I also delete in my current PR, because this code is not necessary. I feel I have a lot of conflicts :)